db/migrate/20160621095352_create_versions.rb

```diff
@@ -0,0 +1,80 @@
+# This migration creates the `versions` table, the only schema PT requires.
+# All other migrations PT provides are optional.
+class CreateVersions < ActiveRecord::Migration
+  # Class names of MySQL adapters.
+  # - `MysqlAdapter` - Used by gems: `mysql`, `activerecord-jdbcmysql-adapter`.
```

Comment:
This file was generated by `paper_trail:install`. I would argue that, if it was not state of the art, the mentioned command would generate the migration without comments.